examples/guides/se-restful-webservice/README.adoc

```diff
@@ -250,208 +200,180 @@ import io.helidon.webserver.ServerRequest;
+respond to an orderly shutdown if you trigger one from your app.
+<4> Report a failed startup.
+--
+. Create routing rules for the app.
 +
 --
 [source,java]
-// _include::0-26:{main-class}[tag=startServer]
+// _include::0-13:{main-class}[tags=createRouting;!addCustomHealthCheck,indent=0]
 ----
-    protected static WebServer startServer() throws IOException {
+private static Routing createRouting(Config config) {
 
-        // load logging configuration
-        LogManager.getLogManager().readConfiguration(
-                Main.class.getResourceAsStream("/logging.properties"));
-
-        // By default this will pick up application.yaml from the classpath
-        Config config = Config.create();
-
-        // Get webserver config from the "server" section of application.yaml
-        ServerConfiguration serverConfig =
-                ServerConfiguration.create(config.get("server")); // <1>
-
-        WebServer server = WebServer.create(serverConfig, createRouting()); // <2>
-
-        // Start the server and print some info.
-        server.start().thenAccept(ws -> { // <3>
-            System.out.println(
-                    "WEB server is up! http://localhost:" + ws.port());
-        });
-
-        // Server threads are not demon. NO need to block. Just react.
-        server.whenShutdown().thenRun(() // <4>
-                -> System.out.println("WEB server is DOWN. Good bye!"));
-
-        return server;
-    }
+    MetricsSupport metrics = MetricsSupport.create();
+    GreetService greetService = new GreetService(config);
+    HealthSupport health = HealthSupport.builder()
+            .add(HealthChecks.healthChecks())   // Adds a convenient set of checks
+            .build(); //<1>
+    return Routing.builder() //<2>
+            .register(JsonSupport.create())
+            .register(health)                   // Health at "/health"
+            .register(metrics)                  // Metrics at "/metrics"
+            .register("/greet", greetService)
+            .build();
+}
 ----
-<1> Gets the webserver config from the "server" section of `application.yaml`. The
-config system automatically maps the config's `host` and `port` to those
-attributes of `ServerConfiguration`.
-<2> Creates the web server using the routing rules from the `createRouting` method.
-<3> Starts the web server, then logs a message.
-<4> Set up a shutdown hook so when the web server is shut down (for example, by ^C on the console)
-the app prints a message.
---
+<1> Sets up several built-in health checks (deadlock, disk space, heap memory) for
+the server.
+<2> Builds the `Routing` instance by registering the JSON, health, metrics, and the
+app's own greeting service.
 
-. Write your main method
-+
---
-Add
-
-[source,java]
-// _include::0-0:{main-class}[tag=mainContent]
-----
-        startServer();
-----
-to your main method.
+Later steps in this guide show how to add your own, app-specific health check and
+metric.
```

Comment:
Should this be "metrics" (plural) ?

Reply:
The tutorial adds one app-specific health check and one app-specific metric, hence singular.